.github/actions/create-azure-resources/action.yaml

```diff
@@ -27,7 +27,11 @@ runs:
     - name: Create Azure resources
       shell: bash
       run: bash ${{ inputs.scripts-setup-dir }}/create_azure_resources_daily.sh
-    
+
+    - name: Setup azcopy
+      shell: bash
+      run: bash ${{ inputs.scripts-setup-dir }}/install_azcopy.sh

```

Comment:
I don't think these should be here, but you could add them to .github\workflows\assets-test.yaml if azcopy will be used to support asset testing.

Reply:
Done